src/app/note/page.tsx

```diff
@@ -0,0 +1,25 @@
+import NoteCanvas from "@/components/notecanvas";
+import useState from 'react';
+
+interface INote {
+  _id: string;
+  title: string;
+  content: string;
+  // authorId: string;
```

Comment:
I guess you don't need this commented out type?
```suggestion

```

Reply:
It may come in handy for authentication and authorization purpose.

Reply:
Makes sense but what I wouldn't have know that so what you could do is to add a comment here like so incase of next time
```suggestion
  // TODO: add `authorId` for authentication later on
```